modules/@apostrophecms/asset/lib/webpack/src/webpack.config.js

```diff
@@ -41,7 +41,9 @@ module.exports = ({
     },
     resolveLoader: {
       extensions: [ '*', '.js' ],
-      modules: [ 'node_modules' ]
+      // Make sure css-loader and postcss-loader can always be found, even
+      // if npm didn't hoist them
+      modules: [ 'node_modules', 'node_modules/apostrophe/node_modules' ]
```

Comment:
The opposite problem. css-loader must be findable by project level even if only apostrophe has it (because npm link was used and hoisting hasn't happened).